### DISCUSSION TITLE
symbol.directory

### DISCUSSION THREAD

**Discussion by aleixmorgadas** [symbol-dapp/community#5] [Dapps Ideas]
A first use case as a Dapp for Symbol I want to create is a List of Projects. I got inspired in:

- https://dappradar.com/
- https://www.dapp.com/

and others.

The idea is to make a similar project but as a Distributed Application and for Symbol related projects.

My idea is that people will be able to:

__Milestone 1__
- Publish their project
- List projects
- Find by Category and other properties

__Milestone 2__
- Rate a project
  - Star rating 1 to 5
  - Text opinion

You can see the progress at https://github.com/symbol-dapp/symbol.directory and the website is https://symbol.directory/

Hopefully, it can be a use case example for other people to get inspiration on how to build a Dapp using symbol-dapp :smile:

**Comment by evias** [comment]
I like to refer at this as a **registry** and the reason I am commenting here is that I would like to discuss the *attributes of a dapp* in this discussion and also I would like to understand their *on-chain footprint*.

To make this proposal compatible with any symbol-dapp requires an agreement on the metadata that is available about dapps, i.e. you can't access symbol.directory without knowing its' URL and/or name, similarly a developer can't *programmatically* publish a new project if they don't know the structure that needs to be taken into account.

- Is there a proposal about how you want to represent such data in the context of Symbol?

**Reply by evias** [reply]
Redirecting the above question to a subsequent focussed discussion board [Distributing symbol-dapp metadata]( https://github.com/symbol-dapp/community/discussions/12)

**Reply by aleixmorgadas** [reply]
The submission form can be found here https://symbol.directory/submit
